client/webserver/site/src/js/register.js

```diff
@@ -26,10 +24,10 @@ export default class RegistrationPage extends BasePage {
       // Form 3: Unlock Decred wallet
       'unlockWalletForm',
       // Form 4: Configure DEX server
-      'dexAddrForm',
+      'dexAddrForm', 'dexAddr',
       // Form 5: Confirm DEX registration and pay fee
       'confirmRegForm', 'feeDisplay', 'dcrBaseMarketName', 'dexDCRLotSize', 'appPass', 'submitConfirm', 'regErr',
```

Comment:
Can clean up some of these.  `dexDCRLotSize` is gone from markets.tmpl now, maybe others.